docs/source/API/core/view/view.md

```diff
@@ -67,22 +68,32 @@ Template parameters other than `DataType` are optional, but ordering is enforced
  *  `memory_space`: Data storage location type. 
  *  `device_type`: the compound type defined by `Device<execution_space,memory_space>`
  *  `memory_traits`: The memory traits of the view. 
- *  `host_mirror_space`: Host accessible memory space used in `HostMirror`.
+ *  `host_mirror_space`: compatible view type with the same `DataType` and `LayoutType` stored in host accessible memory space.
+ * `uniform_type`: this view type with all template arguments explicitly defined, and brought into a canonical form.
+   * e.g. 0D and 1D `LayoutLeft` and `LayoutRight` are mapped to the same layout type, and always uses `device_type` as space argument.
+ * `uniform_const_type`: `uniform_type` with const data type
+ * `uniform_runtime_type`: as `uniform_type` but without compile time extents.
+ * `uniform_runtime_const_type`: `uniform_runtime_type` with const data type.
+ * `uniform_nomemspace_type`: uses unified layout and `AnonymousSpace`.
+ * `uniform_const_nomemspace_type`: `uniform_nomemspace_type` with const data type.
+ * `uniform_runtime_nomemspace_type`: uses unified layout, all runtime extents and `AnonymousSpace`.
+ * `uniform_runtime_const_nomemspace_type`: `uniform_runtime_nomemspace_type` with const data type.
```

Comment:
are all these new above supposed to be under "spaces"?  or should be under "viewTypes" ?